src/modules/pages/components/RouteLayouts/UserNavigation.css

```diff
@@ -190,3 +190,48 @@
   vertical-align: top;
   font-size: var(--size-tiny);
 }
+
+.networkHealth {
+  composes: flexContainerColumn from '~styles/layout.css';
+  margin-top: 17px;
+  margin-right: 10px;
+  margin-left: 10px;
+  height: 124px;
+  width: 352px;
+  background-color: var(--grey-blue-2);
+}
+
+.networkHealthHeading {
+  font-size: var(--size-smallish);
+  font-weight: var(--weight-bold);
+  line-height: 18px;
+  color: var(--grey-1);
+}
+
+.networkHealthDescription {
+  margin-top: 9px;
+  font-size: var(--size-normal);
+}
+
+.link {
+  display: flex;
+  justify-content: right;
+  margin-top: 12px;
+  font-size: var(--size-small);
+  font-weight: var(--weight-bold);
+  color: var(--action-secondary);
+}
+
+.networkHealthIcon {
+  margin-right: 8px;
+}
+
+.networkHealthIcon-critical svg {
```

Comment:
I'm interested why you opted for kebab-case? I thought we should use only camelcase? (even though the *.css.d.ts generates camelcase)

Reply:
I saw other instances of kebab-case in the codebase. For example - https://github.com/joincolony/colonyDapp/blob/41f9ed47667ca73822740cf4e04b88ed0f4b00f5/src/modules/core/components/OmniPicker/OmniPicker.tsx#L227 .

Reply:
👍 I'll try to use it if an appropriate situation arises.